Jenkinsfile

```diff
@@ -25,8 +25,9 @@ def startLisk() {
 	try {
 		sh '''#!/bin/bash
 		cp test/config.json test/genesisBlock.json .
+		sed -i 's#95#200#g' config.json
```

Comment:
this change will currently increase a poolSize from 95 to 200. Why do we need this?